qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -48,9 +42,6 @@ def convert_to_target(
     configuration: BackendConfiguration,
     properties: BackendProperties = None,
     defaults: PulseDefaults = None,
-    *,
-    include_control_flow: bool = True,
```

Comment:
Please make sure this function is not used in the internal software stack.

Reply:
Hmm, unfortunately, I found that, although this is an undocumented utility function, it is used a few times in the internal software stack. Perhaps it should be documented as a public function.

Luckily, no internal usage of this function references `include_control_flow` or `include_fractional_gates` and the removal of the options here preserves the default behavior so there should be not disruption to the internal usage.

My reasoning was that the filtering of the raw API data was sufficient. There was no need to filter a second time during target creation. Since `convert_to_target` was not documented, I thought I was free to remove the extra options and filtering. I don't feel too strongly though. I can put them back if that seems best.

Reply:
I think you can at least add warning for usage of these options. But I'm also fine removing them as long as these options are not found by search (since key is required).

Reply:
I decided just to restore the options to `convert_to_target`. Since `IBMBackend` filters the API data, I did not restore passing the options to `converet_to_target` from it. Passing the options would only matter if `backend.options.use_fractional_gates` were modified but with the caching of the configuration/properties/target the results would be inconsistent unless `refresh()` were used but then the target would be consistent again without passing the options.